### ISSUE TITLE
When adding recipients, user picker does not find users with apostrophe

### ISSUE THREAD

**Issue by funkycram** [humhub/mail#245]
Humhub v1.10 - Mail v2.0.7

Related to https://github.com/humhub/humhub/issues/5359

### What steps will reproduce the problem?

Rename the firstname (or lastname) of a user to `David d'Auterive Roberts`.
Create a new message.
In the "Add recipients" field, type `d'Auterive`

### What is the expected result?

No result

### What do you get instead?

Should find this user.

**Comment by luke-** [comment]
@funkycram Is this bug new with HumHub v1.10+?

**Comment by funkycram** [comment]
No, but as Humhub 1.10 has fixed the apostrophe problem in the user picker, I have mentioned that the bug is still present with this version of Humhub. But the bug was already present before.

**Comment by funkycram** [comment]
Tested. It works now. Thanks!
